<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcpCli</class>
 <widget class="QDialog" name="tcpCli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tcpCli</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditServerIP"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonSend">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditPort"/>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>服务器地址</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>端口号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditUsername"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonEnter">
      <property name="text">
       <string>进群</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
